File: 02-EstruccturasDeControl/programaJava/JuegoPreguntas.java
import javax.swing.JOptionPane;
/*
    Universidad Tecnologica de Tula-Tepeji
    Programa educativo: Tecnologias de la Informacion e innovacion
    TSU. Desarrollo de Software Multiplataforma
    Programador: Armandoo Emmanuel Gomez Almaraz
    "Fecha de creacion: 22/01/2026"
    Descripcion: en un juego de prenguntas als que se responde si o no gana quien responta correctamente las 3 preguntas.
    Si responde mal a cualquiera de ellas ya no se prengunta la siguente y termina el juego
*/
public class JuegoPreguntas {
    public static void main(String[] args) {
        // Se da Una bienvenida al juego
        JOptionPane.showMessageDialog(null, "Responde las siguientes preguntas con SI o NO");
        // Primera pregunta
        String respuesta1 = JOptionPane.showInputDialog("EL primer albun de Zoe es Demo Olmos?");
        // Se utiliza un switch para evaluar la respuesta
        switch (respuesta1) {
            case "SI":
                // Segunda pregunta
                String respuesta2 = JOptionPane.showInputDialog("El segundo albun de Zoe es Karmatica Resonancia?");
                switch (respuesta2) {
                    case "SI":
                       JOptionPane.showMessageDialog(null, "Respuesta incorrecta, fin del juego");
                        
                        break;
                    case "NO":
                         // Tercera pregunta
                        String respuesta3 = JOptionPane.showInputDialog("El tercer albun de Zoe es Aztlan?");
                        switch (respuesta3) {
                            case "SI":
                                 JOptionPane.showMessageDialog(null, "Respuesta incorrecta, fin del juego");
                                break;
                            case "NO":
                                JOptionPane.showMessageDialog(null, "Felicidades, has respondido correctamente las 3 preguntas y has ganado el juego");
                                break;
                            default:
                                break;
                        }
                        break;
                    default:
                        break;
                }
                break;
            case "NO":
                JOptionPane.showMessageDialog(null, "Respuesta incorrecta, fin del juego");
                break;
                
            default:
                break;
        }
    }
}

File: 02-EstruccturasDeControl/programaPython/JuegoPreguntas.py
import tkinter as tk
from tkinter import messagebox

def juego_preguntas():
    # Se da una bienvenida al juego
    messagebox.showinfo("Bienvenido", "Responde las siguientes preguntas con SI o NO")

    # Primera pregunta
    respuesta1 = messagebox.askquestion("Pregunta 1", "¿El primer álbum de Zoe es Demo Olmos?")
    
    if respuesta1 == "yes":
        # Segunda pregunta
        respuesta2 = messagebox.askquestion("Pregunta 2", "¿El segundo álbum de Zoe es Karmatica Resonancia?")
        
        if respuesta2 == "yes":
            messagebox.showinfo("Fin del juego", "Respuesta incorrecta, fin del juego")
        else:
            # Tercera pregunta
            respuesta3 = messagebox.askquestion("Pregunta 3", "¿El tercer álbum de Zoe es Aztlan?")
            
            if respuesta3 == "yes":
                messagebox.showinfo("Fin del juego", "Respuesta incorrecta, fin del juego")
            else:
                messagebox.showinfo("Ganaste", "Felicidades, has respondido correctamente las 3 preguntas y has ganado el juego")
    else:
        messagebox.showinfo("Fin del juego", "Respuesta incorrecta, fin del juego")

# Configuración de la ventana principal
root = tk.Tk()
root.withdraw()  # Oculta la ventana principal
juego_preguntas()